lib/strategy.ex

```diff
@@ -18,6 +18,9 @@ defmodule LibclusterPostgres.Strategy do
   alias Cluster.Strategy
   alias Cluster.Logger
 
+  @postgrex_keys [:hostname, :username, :password, :database, :port] ++
+                   [:ssl, :ssl_opts, :socket_options, :parameters]
```

Comment:
```suggestion
                   [:ssl, :ssl_opts, :socket, :socket_dir, :socket_options, :parameters]
```

Is there any reason not to include `:socket` and `:socket_dir` in the options that get passed to Postgrex?

We would really like to use this at our company, but we use a cloud sql proxy with `:socket_dir` so `:hostname` only will not work for us.

Reply:
🙏 Thank you 🙏